apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -768,29 +768,96 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
            end)
   end
 
+  describe "available piggybacks" do
+    @tag fixtures: [:alice, :processor_filled, :transactions]
+    test "detects available piggybacks because txs not seen in valid block",
+         %{alice: alice, processor_filled: processor, transactions: [tx1, tx2]} do
+      exit_processor_request = %ExitProcessor.Request{
+        blknum_now: 5000,
+        eth_height_now: 5
+      }
+
+      %Transaction{outputs: [%{owner: tx1_owner1}, %{owner: tx1_1owner2}]} = tx1
+
+      event_1 =
+        prepare_piggyback_available_event(
```

Comment:
after you handle the non-zero ones. I think instead of the `prepare...` function you can DRY `txbytes = Transaction.encode(tx1)` and you should be able to assert without `event_1` variable. Also I think you can afford to minimize the assertion slightly. What is most important to assert in this test, is that:
1/ there are two piggybacks available, for different txbytes
2/ owners and indices make sense **for one of the events**